packages/rlc-common/src/buildObjectTypes.ts

```diff
@@ -40,6 +48,19 @@ export function buildObjectInterfaces(
     ) {
       continue;
     }
+
+    // FIXME: disabling new multipart generation for modular while we figure out the story
+    if (objectSchema.isMultipartBody && !model.options?.isModularLibrary) {
```

Comment:
nit: what RLC version will be generated in modular SDK? I image that new design would align with v2, old design would align with v1.

Reply:
We just generate a normal RLC model in the modular SDK (as if it was a plain old JSON operation) because of this check. This is more in line with v1 of the Core package. We can't generate the v2 models here yet as that results in a compile error, since the serializers don't serialize into the array shape.